dripline/extensions/thermo_fisher_endpoint.py

```diff
@@ -0,0 +1,82 @@
+from dripline.core import Entity, calibrate, ThrowReply
+
+import logging
+logger = logging.getLogger(__name__)
+
+__all__ = []
+__all__.append("ThermoFisherGetHexEntity")
+__all__.append("ThermoFisherGetEntity")
+
+
+class ThermoFisherGetHexEntity(Entity):
+    '''
+    A simple endpoint which stores a value but returns that value + a random offset
+    '''
+
+    def __init__(self, 
+                 get_str=None,
+                 **kwargs):
+        '''
+        Args:
+            cmd_str: hexstring of the command, e.g. 20 
+        '''
+        if get_str is None:
+            raise ValueError('<base_str is required to __init__ ThermoFisherGetEntity instance')
+        else:
+            self.cmd_str = str(get_str).zfill(2)
+        Entity.__init__(self, **kwargs)
+
+    @calibrate()
+    def on_get(self):
```

Comment:
I have a question on the `@calibrate` qualifier on the `on_get` method. At first I tried to call the `ThermoFischerHexGetEntity.on_get()` function within the `ThermoFischerNumericGetEntity.on_get()`, since we first have to get the hex string as in the ThermoFischerHexGetEntity and then do further work on it. However when running this code, I got errors from the calibration method, which tried to do the calibration on the result of `ThermoFischerHexGetEntity.on_get()` instead of the `ThermoFischerNumericGetEntity.on_get()`. Because of that I fully rewrote this piece. Any suggestions if that can be simplified?